src/main/java/com/google/sps/NodeMutationFilterTest.java

```diff
@@ -74,7 +69,7 @@ public void getMutationsOfBasic() {
 
   /** Test that a null query returns an empty list */
   @Test
-  public void getMutationsOfNull() {
+  public void getMutationsOfNullName() {
     Mutation addAB =
         Mutation.newBuilder()
             .setType(Mutation.Type.ADD_EDGE)
```

Comment:
It might be nice to add a comment before the `getMutationsOfMultiple()` function to indicate we're testing a different function from the tests above.